app/views/casework/crime_applications/_client_other_charge.html.erb

```diff
@@ -0,0 +1,30 @@
+<% title_key = case_details.partner_other_charge_in_progress.present? ? :client_other_charges_explicit : :client_other_charges %>
+
+<%= govuk_summary_card(title: label_text(title_key)) do
```

Comment:
These partials could be refactored into one shared card with all the variables passed into it